src/components/claiming/Loading.vue

```diff
@@ -40,6 +37,7 @@ export default {
   },
   watch: {
     async sessionId(value) {
+      console.log(value)
```

Comment:
Remove log

Reply:
Would love if we could automate this :sweat_smile:

Reply:
https://eslint.org/docs/rules/no-console.html